docs/404.md

```diff
@@ -0,0 +1,3 @@
+---
+layout: 404
```

Comment:
does the `docs` dir already go to top level? do we need to add a permalink so that it's in the right place to be a 404 page for CG pages?
```suggestion
layout: 404
permalink: /404.html
```

Reply:
I think it's fine without? It seems to be [available at `/404.html`](https://federalist-2f194a10-945e-4413-be01-46ca6dae5358.sites.pages.cloud.gov/preview/18f/identity-style-guide/aduth-docs-404/404.html) already without specifying `permalink`.